### PR TITLE
RIA-3651 update ftpa tabs when decision set aside and reheard

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3651](https://tools.hmcts.net/jira/browse/RIA-3651)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -88,6 +88,7 @@ featureFlag:
   isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:false}
   isSaveAndContinueEnabled: ${IS_SAVE_AND_CONTINUE_ENABLED:true}
   isHomeOfficeIntegrationEnabled: ${IA_HOME_OFFICE_INTEGRATION_ENABLED:false}
+  isReheardAppealEnabled: ${IA_REHEARD_APPEAL_ENABLED:false}
```
Comment (makowalski) [inline src/main/resources/application.yaml:91]: can we use Launch Darkly flag system for it? F&P and HO go old way but new epics should go with Launch Darkly.

**Review by makowalski — DISMISSED** [review]
LGTM

File: src/main/resources/application.yaml (hunk 1 above)
Comment (bradsorour) [reply src/main/resources/application.yaml]: This is now in place.

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Pull request merged**